pkg/v1/mutate/mutate.go

```diff
@@ -113,6 +113,29 @@ func Config(base v1.Image, cfg v1.Config) (v1.Image, error) {
 	return ConfigFile(base, cf)
 }
 
+// ManifestAnnotations mutates the provided v1.Image to have the provided annotations
+func ManifestAnnotations(base v1.Image, annotations map[string]string) (v1.Image, error) {
+	m, err := base.Manifest()
+	if err != nil {
+		return nil, err
+	}
+
+	if len(m.Annotations) < 1 {
```

Comment:
Check for `nil` instead?

Reply:
yes, it should be like this way, it is my bad, thank you 🙏